vuln/SCHEMA.md

```diff
@@ -0,0 +1,42 @@
+# Schema for vulnerabilities
+
+| Key | Type | Description | Example |
+|-----|------|-------------|---------|
+|`id`|Integer > 0|Unique number identifying the record|349|
+|`created_at`|date-time|Record creation|"2017-05-19T22:45:59.16+00:00"|
+XXX do we need this, or is it just a mongo internal field?
+
+|`updated_at`|date-time|Record update|"2017-06-05T13:01:57.071+00:00"|
+XXX do we need this, or is it just a mongo internal field?
+
+|`title`|String|Short readable description|"Directory Traversal"|
+|`author`|String|Finder of vuln|"Liang Gong"|
+XXX what if reporter was not finder?
```

Comment:
+1 to changing this name, we never did because of legacy support but yes I'd change it and possibly add other fields to give appropriate credit such as who authored the advisory, who found the issue, etc.